src/app/pages/StoryPage/index.test.jsx

```diff
@@ -274,15 +274,14 @@ describe('Story Page', () => {
   });
 
   it.each`
-    environment | showAdsBasedOnLocation | showAdsBasedOnLocationExpectation
-    ${'live'}   | ${true}                | ${'permitted to be shown'}
-    ${'test'}   | ${true}                | ${'permitted to be shown'}
-    ${'live'}   | ${false}               | ${'not permitted to be shown'}
-    ${'test'}   | ${false}               | ${'not permitted to be shown'}
+    showAdsBasedOnLocation | showAdsBasedOnLocationExpectation
+    ${true}                | ${'permitted to be shown'}
+    ${true}                | ${'permitted to be shown'}
+    ${false}               | ${'not permitted to be shown'}
```

Comment:
We can probably remove these duplicate lines now

Reply:
Oops, just seen this. I'll do this in a separate PR